modules/text_server_adv/SCsub

```diff
@@ -25,6 +25,9 @@ if "svg" in env.module_list:
     # Enable ThorVG static object linking.
     env_text_server_adv.Append(CPPDEFINES=["TVG_STATIC"])
 
+if env["builtin_icu4c"]:
+    env_text_server_adv.Append(CCDEFINES=["HAVE_ICU_BUILTIN"])
```

Comment:
```suggestion
    env_text_server_adv.Append(CPPDEFINES=["HAVE_ICU_BUILTIN"])
```
I made a typo in the patch.

Reply:
Ah, right. Makes sense why that didn't affect my Debian build...